packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -579,17 +605,21 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   }
 
   private _isImmediateDescendantOfZone(element?: HTMLElement): boolean {
+    return this._getOwnerZone(element) === this.refs.root;
+  }
+
+  private _getOwnerZone(element?: HTMLElement): HTMLElement {
     let parentElement = getParent(element);
 
     while (parentElement && parentElement !== this.refs.root && parentElement !== document.body) {
       if (isElementFocusZone(parentElement)) {
-        return false;
+        return parentElement;
       }
 
       parentElement = getParent(parentElement);
     }
 
-    return true;
+    return this.refs.root;
```

Comment:
is it possible that the element passed in to this method is not attached to the dom, or is not a child of this zone? If so it may not make sense to return this.refs.root.